packages/next-auth/src/providers/ping-id.js

```diff
@@ -0,0 +1,48 @@
+/** @type {import(".").OAuthProvider} */

```

Comment:
Could you revert this please? `next-auth` does not accept new providers at this point. :pray:

Reply:
I reverted it